dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/MetadataSchemaRestRepository.java

```diff
@@ -73,4 +87,98 @@ public MetadataSchemaRest findOne(Context context, Integer id) {
     public MetadataSchemaResource wrapResource(MetadataSchemaRest bs, String... rels) {
         return new MetadataSchemaResource(bs, utils, rels);
     }
+
+    @Override
+    @PreAuthorize("hasAuthority('ADMIN')")
+    protected MetadataSchemaRest createAndReturn(Context context)
+            throws AuthorizeException, SQLException {
+
+        // parse request body
+        MetadataSchemaRest metadataSchemaRest;
+        try {
+            metadataSchemaRest = new ObjectMapper().readValue(
+                    getRequestService().getCurrentRequest().getHttpServletRequest().getInputStream(),
+                    MetadataSchemaRest.class
+            );
+        } catch (IOException excIO) {
+            throw new PatchBadRequestException("error parsing the body ..." + excIO.getMessage());
```

Comment:
Again, swallowing/hiding the stacktrace.  This should be:
`throw new PatchBadRequestException("error parsing the body", excIO);`